### PR TITLE
add login

### PR DESCRIPTION
**Pull request by Ayushi20-19**
<h1>FEATURE ADDED</h1>
<ul>
<li>add login page</li>
</ul>

<h1>FILE TO CHECK</h1>
<ul>
<li>login.js</li>
</ul>

<b> link </b> https://winter-store-git-authpage-ayushi20-19.vercel.app/
<b>All the form error handling will be add later, you can skip CSS files only functionality needs to be reviewed.</b>

### CODE REVIEW

File: src/components/Auth/Login.js [hunk 1]
```diff
@@ -1,6 +1,53 @@
...
+    setUserDetail(testUser);
+  };
+  const submitUserDetail = async (e) => {
+    e.preventDefault();
+    try {
+      const response = await axios.post("/api/auth/login", {
+        email: userDetail.email,
+        password: userDetail.password,
+      });
+
```
Comment (starver20) [inline src/components/Auth/Login.js:35]: You can add a check here to see if the response status is 200, only then you shloud continue with rest of the operations.

File: src/components/Auth/Signup.js [hunk 2]
```diff
@@ -14,18 +14,21 @@ const Signup = () => {
...
 
   const submitUserDetail = async (e) => {
     e.preventDefault();
     if (userDetail.password === userDetail.confirmPassword) {
       try {
-        const response = await axios.post(`/api/auth/signup`, userDetail);
+        const response = await axios.post("/api/auth/signup", {
+          email: userDetail.email,
+          password: userDetail.password,
+        });
```
Comment (starver20) [inline src/components/Auth/Signup.js:31]: Same response status check is needed here

File: src/components/Auth/Signup.js [hunk 3]
```diff
@@ -14,18 +14,21 @@ const Signup = () => {
...
         authDispatch({
           type: "SIGN_UP",
           payload: {
```
Comment (starver20) [inline src/components/Auth/Signup.js:34]: I believe you are going to add the payload later

**Review by starver20 — COMMENTED** [review]
other than the suggestions rest of the code looks good.

**Review by AnkurChunekar — CHANGES_REQUESTED** [review]
As suggested by @starver20 for better error handling response status should be checked if it's same as expected, other than that code looks good to me.

**Pull request merged**